haiku-apps/bamkeys/bamkeys-1.0.0.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="An input_server filter for Haiku that \
+lets you spatially map your display for navigating \
+windows with your keyboard"
+DESCRIPTION="BamKeys is an input_server filter for \
+Haiku that lets you spatially map your display for \
+navigating windows with your keyboard. BamKeys let's you \
+break your display up into a grid of zones. Each zone \
+is assigned a keyboard combination. Whenever you press \
+that keyboard combination, BamKeys will focus \
+the front-most window that intersects with that zone, \
+regardless of the program it belongs to! Pressing the \
+same zone's keyboard combination repeatedly will focus \
+windows further back. If you have three windows that \
+intersect a zone, pressing that zones hotkey three times \
+will cycle through all those windows."
```

Comment:
Since a Help file doesn't seem to be included (maybe that could be added in a PR to Brian as well, best reachable from the BamKeys panel), you should add the instructions from the ReadMe to the Description.

Reply:
No, no. :) Just what's under "Configuring". Maybe you can condense it a bit...